Soccar [UI]/src/nl/socnet/message/ChatMessage.java

```diff
@@ -13,24 +13,24 @@
 @MessageEvent(id = MessageConstants.CHAT_MESSAGE_ID, handler = ChatMessageHandler.class)
 public final class ChatMessage extends Message {
 
-    private final String username;
+    private final int playerId;
     private final Privilege privilege;
     private final String message;
 
-    public ChatMessage(String username, Privilege privilege, String message) {
-        this.username = username;
+    public ChatMessage(int playerId, Privilege privilege, String message) {
+        this.playerId = playerId;
         this.privilege = privilege;
         this.message = message;
     }
 
     public ChatMessage(String message) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)